docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Actually, this is just a pile of proposals, a number of which have been made before. Please just file area/usability issues for the new proposals and a separate doc for using the current functionality provided by kubectl.